crates/utils/src/rate_limit/rate_limiter.rs

```diff
@@ -266,66 +345,65 @@ mod tests {
     ];
     for ip in ips {
       let ip = ip.parse().unwrap();
-      let message_passed =
-        rate_limiter.check_rate_limit_full(super::RateLimitType::Message, ip, 2, 1, now);
-      let post_passed =
-        rate_limiter.check_rate_limit_full(super::RateLimitType::Post, ip, 3, 1, now);
+      let message_passed = rate_limiter.check(ActionType::Message, ip, now);
+      let post_passed = rate_limiter.check(ActionType::Post, ip, now);
```

Comment:
Would it also be possible to add a "fail" test in here?

Reply:
Done